keel-sql/src/main/resources/db/changelog/20200925-environment-related-indices.yml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+- changeSet:
+    id: environment-related-indices
+    author: fletch
+    changes:
+    - createIndex:
+        indexName: artifact_versions_name_type_release_status_idx
+        tableName: artifact_versions
+        columns:
+        - column:
+            name: name
+        - column:
+            name: type
+        - column:
+            name: release_status
```

Comment:
See query in `SqlArtifactRepository.versions` here https://github.com/spinnaker/keel/blob/master/keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt#L230-L232

Reply:
I've introduced a similar query in my last PR that also includes `version`. You might want to include that in the same index?

https://github.com/spinnaker/keel/blob/c3b40de05e19562fadd8ec3d6636e4f84f568e6e/keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt#L283-L286